OSPSuite.Presentation.Importer/Core/IDataFormat.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+using OSPSuite.Presentation.Importer.Core.DataFormat;
+using System.Collections.Generic;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+
+   public interface IDataFormat
+   {
+      string Name { get; }
+      string Description { get; }
+      bool CheckFile(IUnformattedData rawData);
+      IList<DataFormatParameter> Parameters { get; }
+      IList<Dictionary<Column, IList<double>>> Parse(IUnformattedData data);
```

Comment:
Not found of this return value. It looks very much low level. 
Also exposing IList assumes that the consumer can modify the content. Why not exposing IReadOnlyList?

Reply:
Maybe we could create a data-structure for this, but I personally do not see a problem in allowing edition from the outside. The point of this method is to create the different datasets, which are actually a copy of the imported unformatted data. What the caller do with the with the data is out of the scope of the format implementation.